source/type/data.cpp

```diff
@@ -62,7 +62,7 @@ namespace pt = boost::posix_time;
 namespace navitia {
 namespace type {
 
-const unsigned int Data::data_version = 69;  //< *INCREMENT* every time serialized data are modified
+const unsigned int Data::data_version = 70;  //< *INCREMENT* every time serialized data are modified
```

Comment:
You do not need to increment `data_version` here, `Impacts` aren't in the `data.nav.lz4` as they are created by kraken at runtime.

Reply:
Done